docs/user-guide/nginx-configuration/configmap.md

```diff
@@ -362,8 +362,10 @@ _References:_
 ## max-worker-connections
 
 Sets the [maximum number of simultaneous connections](http://nginx.org/en/docs/ngx_core_module.html#worker_connections) that can be opened by each worker process.
-The default of 0 uses the value of [max-worker-open-files](#max-worker-open-files).
-_**default:**_ 0
+_**default:**_ 16384
```

Comment:
I'd suggest keeping an explanation on what 0 means, even if it's not the default anymore:

```suggestion
0 will use the value of [max-worker-open-files](#max-worker-open-files).
_**default:**_ 16384
```

Reply:
Good idea. done